src/main/java/handlers/QueueHandler.java

```diff
@@ -66,6 +66,8 @@ private void initializeQueue(){
             //sharing connection between threads
             factory = new ConnectionFactory();
             factory.setHost("localhost");
+            factory.setUsername("rabbitmq");
```

Comment:
I set those values according to the docker compose definitions. Ideally, we should retrieve them from a config file.